servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java

```diff
@@ -26,5 +26,11 @@
  * <p>In particular, this list is about Nessie model classes.
  */
 @RegisterForReflection(
-    targets = {NessieError.class, CELUtil.OperationForCel.class, CELUtil.ContentForCel.class})
+    targets = {
+      NessieError.class,
+      CELUtil.KeyedEntityForCel.class,
+      CELUtil.OperationForCel.class,
+      CELUtil.ContentForCel.class,
+      CELUtil.KeyEntryForCel.class
```

Comment:
seems like we could prevent this from happening in the future by automatically labeling PRs when one of these files changes i.e. `CELUtil.java`...
similar to:
https://github.com/apache/iceberg/blob/master/.github/labeler.yml

wdyt?

Reply:
SGTM :+1: :)

Reply:
#4931